wolfcrypt/src/ecc.c

```diff
@@ -88,10 +88,14 @@ ECC Curve Sizes:
 #endif
 
 /* Make sure ASN is enabled for ECC sign/verify */
-#if (defined(HAVE_ECC_SIGN) || defined(HAVE_ECC_VERIFY)) && defined(NO_ASN)
+#if (defined(HAVE_ECC_SIGN) && defined(NO_ASN))
     #error ASN must be enabled for ECC sign/verify
```

Comment:
Since we have NO_ASN wrappers below around the sign_hash and verify_hash I am not sure this section is even required.

Reply:
Removed the section entirely